packages/components/src/components/ExpandableContainer.tsx

```diff
@@ -32,6 +33,9 @@ export class ExpandableContainer extends React.PureComponent<Props, State> {
     }
 
     handleClick = () => {
+        if (this.props.onClick)
+            this.props.onClick(!this.state.visible);
+
         this.setState(state => ({ visible: !state.visible }));
```

Comment:
I suppose it is technically possible for the value of the `visible` var to be different between line 37 and 39 here. Suggest moving this to the callback of setState:
```suggestion
        this.setState(state => ({ 
             visible: !state.visible 
        }), () = > {
            if (this.props.onClick)
                this.props.onClick(this.state.visible);        
        });
```